app/allocation/controllers/assign/save.test.js

```diff
@@ -0,0 +1,113 @@
+const moveService = require('../../../../common/services/move')
+const personService = require('../../../../common/services/person')
+const CreateSave = require('../../../move/controllers/create/save')
+
+const MixinProto = CreateSave.prototype
+const AssignBaseController = require('./base')
+const SaveController = require('./save')
+
+const controller = new SaveController({ route: '/' })
+const ownProto = Object.getPrototypeOf(controller)
+
+describe('Assign controllers', function() {
+  describe('Assign save controller', function() {
+    it('should extend AssignBaseController', function() {
+      expect(Object.getPrototypeOf(ownProto)).to.equal(
+        AssignBaseController.prototype
+      )
+    })
+
+    describe('When mixing in create controller', function() {
+      it('should copy process from CreatePersonSearch', function() {
+        expect(controller.process).to.exist.and.equal(MixinProto.process)
+      })
+
+      it('should only have the expected methods of its own', function() {
+        const ownMethods = ['saveValues', 'errorHandler', 'successHandler']
```

Comment:
There only seem to be tests for `saveValues()`, can you add tests for the `errorHandler` and `successHandler` while you're here please

Reply:
Added